packages/kerosene/src/datetime/index.ts

```diff
@@ -34,3 +34,15 @@ export enum DayOfWeek {
   FRIDAY,
   SATURDAY,
 }
+
+export { default as getCalendarWeeks } from "./getCalendarWeeks";
+import {
+  CalendarWeek as _CalendarWeek,
+  CalendarWeeks as _CalendarWeeks,
+} from "./getCalendarWeeks";
+export type CalendarWeek = _CalendarWeek;
```

Comment:
can you explain this pattern?

Reply:
Because we use `babel` to transpile, it isn't aware that `CalendarWeek` is a type if we just did `export { default as getCalendarWeeks, CalendarWeek, CalendarWeeks } from "./getCalendarWeeks";`.
When re-exporting types, you need to either do `export * from ...;` or be explicit by doing `export type` so that `babel` doesn't choke.